go/login/server/example-server/cmd/run.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"../../../../glome"
+	"../../../server"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	keys        string
+	addr        string
+	script      string
+	userHeader  string
+	responseLen int
+
+	runCmd = &cobra.Command{
+		Use:   "run [-a <addr>] [-k <keyfile>] [-s <lua-script>]",
+		Short: "Run command creates a LoginServer that listen and serve in provided address.",
+		Long: `Run implements the server side of the glome-login protocol.
+Visit github.com/google/glome for more information on the protocol.
+
+This program implements the basic functionalities of the glome-server, using the glome-framework.
+It requires the flags -a, -k and -s to be provided to properly work. For more information, visit [add link to README.md]`,
```

Comment:
Link is still to be added, but I'm not sure whether there's more to be learned in the readme :)

Reply:
Deleted